nexus/db-queries/src/db/datastore/instance.rs

```diff
@@ -385,12 +386,14 @@ impl DataStore {
         new_instance: &InstanceRuntimeState,
         vmm_id: &Uuid,
         new_vmm: &VmmRuntimeState,
-    ) -> Result<(bool, bool), Error> {
+        migration: &Option<MigrationRuntimeState>,
+    ) -> Result<(bool, bool, Option<bool>), Error> {
```

Comment:
This changes makes the `Return value` comment out-of-date -- could we use a named type here instead?